.github/workflows/bump_submodules.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Submodule
```

Comment:
Please add a descriptive comment to this top of this file. You can also add a license header (see other .yml files in this directory)